Examples/Python/tests/test_alignmentdecorator.py

```
@@ -0,0 +1,85 @@
+        numThreads=1,
+        logLevel=acts.logging.INFO,
+    )
+
+    sequencer.addContextDecorator(alignDeco)
+    sequencer.run()
+    # Count that the alignment store is decorated 37 times
+    out, err = capfd.readouterr()
+    assert out.count("Decorating AlgorithmContext with alignment store") == 37
+
+
+test_alignmentdecorator_gen_mode()
```

Pylint (3.3.7)

[error] No value for argument 'capfd' in function call

(E1120)